.env.example

```diff
@@ -0,0 +1,15 @@
+# USE THIS FILE AS A TEMPLATE TO CREATE YOUR OWN .env FILE
+AIRTABLE_PAT = "" # personal acess token for airtable
+AIRTABLE_BASE_ID = "appr8SepSgx9SP6ir"
+AIRTABLE_CONTACTS = "tblXBR1YTXYvGhwWJ"
+AIRTABLE_EMPLOYEESandVOLUNTEERS = "tblk3Qd3sKV9JPNtx"
+AIRTABLE_SERVICE = "tbl2hG3F6LlSW3Bm7"
+AIRTABLE_BIKESforSALE = "tblWAo0tdOEFN2RaA"
+AIRTABLE_BIKESforDONO = "tblGlO7tDFuQg3RKy"
+AIRTABLE_RENTALBIKES = "tbllDlYHfpxvWi5XH"
+AIRTABE_WEBSHOPINQUIRY = "tblaKhFw5X6cvBNNp"
+AIRTABLE_SALESORDERS = "tblp14q73VINEXGM1"
+AIRTABLE_ORDERITEMS = "tblCKoA3zEjflGoLf"
+AIRTABLE_INTERACTIONS = "tblYDL11X6PuvlnKA"
+AIRTABLE_NOLONGERUSERSALES = "tbldLNHp78lSBKaxZ"
+AIRTABLE_RENTAL = "tblATiIu8nRsN9JSW" 
```

Comment:
I liked the idea you had of adding these to the config instead, I think that would be better. We should leave the TEST keyword in, and just add an underscore BETWEEN_EACH_WORD_LIKE_THIS. The reason is that it is more readable that way and we are accessing a test database after all. Once we start actually deploying and they use the program on their actual database we can create new constants for the real deal.

Reply:
oh my bad I thought that was the real database